one.alynx.FlipClock.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: one.alynx.FlipClock
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: flipclock
+modules:
+  # The SDL_ttf in `shared-modules` repo is for SDL1,
+  # and the SDK only contains SDL2 but has no SDL2_ttf,
+  # so I will build and bundle SDL2_ttf by myself.
+  - name: SDL2_ttf
```

Comment:
2.0.15 is in the SDK. Isn't it enough?

Reply:
Sorry, don't know why I never see it yesterday.

Reply:
Please check my newest commit.